mockgcp/mockcloudidentity/groups.go

```diff
@@ -154,7 +154,7 @@ func (s *groupsServer) PatchGroup(ctx context.Context, req *pb.PatchGroupRequest
 	for _, path := range strings.Split(req.GetUpdateMask(), ",") {
 		switch path {
 		case "displayName": // TF controller uses displayName while direct controller uses display_name
-			obj.DisplayName = req.GetGroup().DisplayName // TF
+			fallthrough
```

Comment:
nit: maybe `case "displayName", "display_name":` would be less error-prone for future edits?